encrypted-media.js

```diff
@@ -409,7 +409,7 @@
           EME_spec_url = prefix + file;
           groupBaseURLs[x] = EME_spec_url;
           // Refer to the Web IDL Editor’s Draft from Editor’s Drafts of this spec.
-          IDL_spec_url = "https://heycam.github.io/webidl/";
+          IDL_spec_url = "https://www.w3.org/TR/WebIDL-1/";
```

Comment:
This URL is just a redirect to the WHATWG spec, better use the final URL directly. W3C no longer publishes the Web IDL spec in particular.

```suggestion
          IDL_spec_url = "https://webidl.spec.whatwg.org/";
```